### ISSUE TITLE
I find this odd

### ISSUE THREAD

**Issue by mmaxen** [AdobeDocs/target-developers#85]
Issue in /src/pages/implement/client-side/atjs/upgrading-from-atjs-1x-to-atjs-20.md

The word Yes is a link next to response tokens but none if the other yeses and nos are linked.  I think it would be better to have the feature named be a link, so for example have the feature named "Response Tokens" take the user to that help page.  

![image](https://user-images.githubusercontent.com/106278998/170369163-7208092b-149f-4c77-a0c1-f0cbc65b52b1.png)

**Comment by judy-kim** [comment]
Agreed! Nice catch, thanks. Fixed this one, plus another similar one on the same page.
